packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,220 @@
+                            Newsroom
+                        </p>
+                    )}
+                </Link>
+                <div
+                    className={twMerge(
+                        'md:items-center justify-between gap-12 md:gap-4 hidden md:flex',
+                        Boolean(openMobileNav && !isSm) &&
+                            `flex flex-col w-screen absolute top-24 left-0 z-10 bg-white border-b border-gray-200`,
+                        layout === 'centered' ? `lg:w-2/3` : 'md:w-max',
+                    )}
+                >
+                    {Boolean(
+                        categories.length ||
+                            hasStandaloneAboutPage ||
+                            hasStandaloneContactsPage ||
+                            publicGalleriesCount,
+                    ) && (
+                        <div className="pt-6 md:pt-0 flex flex-col md:flex-row md:items-center gap-12 md:gap-4 px-6 md:px-0">
+                            {categories.length > 0 && (
+                                <CategoriesDropdown categories={categories} locale={locale} />
+                            )}
+                            {publicGalleriesCount > 0 && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
+                                    href="/media"
+                                    locale={locale ?? false}
+                                >
+                                    {/* TODO: Use translations */}
+                                    Media
+                                </Link>
+                            )}
+                            {hasStandaloneAboutPage && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
+                                    href="/about"
+                                    locale={locale ?? false}
+                                >
+                                    {/* TODO: Use translations */}
+                                    About
+                                </Link>
+                            )}
+                            {hasStandaloneContactsPage && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
```

Comment:
This class names construction is re-used multiple times, so probably better to extract the link into a component, or compute the class name once somewhere up in the code.